webapp/models.py

```diff
@@ -125,6 +140,23 @@ def as_json(self):
         }
 
 
+class Salesforce_Campaign(DateTimeMixin):
+    __tablename__ = "salesforce_campaign"
+    name = Column(String, nullable=True)
+    id = Column(String, primary_key=True)
+    assets = relationship(
+        "Asset",
+        secondary=asset_campaign_association_table,
+        back_populates="salesforce_campaigns",
+    )
+
+    def as_json(self):
```

Comment:
Not sure if it matters, but this function does not create a JSON object, it creates a Python dict

Reply:
Yeah, I just followed the already existing naming conventions.